specification/attributes/column_naming_and_ordering_convention.md

```diff
@@ -1,23 +1,24 @@
-# Column Naming Convention
+# Column Naming and Ordering Convention
```

Comment:
nit: Should we remove "Convention"? "Naming convention" makes sense, but "ordering convention" doesn't seem right. Given it's an attribute, it feels unnecessary. I'm fine either way. Just a nit.
```suggestion
# Column Naming and Ordering
```